bioptim/optimization/solution.py

```diff
@@ -1021,6 +1021,25 @@ def print_constraints(ocp, sol):
                 print("")
             print(f"------------------------------")
 
+        def print_parameters(ocp, sol):
+            """
+            Print the values of each optimized parameters
+            """
+
+            if sol.parameters['all'].size == 0:
```

Comment:
That may be the case, but I am not sure. Did you confirm that there actually is a "all" when no parameters are defined?

Reply:
Yes, I do.